pkg/antctl/raw/helper.go

```diff
@@ -220,3 +225,26 @@ func CreateControllerClientCfg(
 	cfg.Host = fmt.Sprintf("https://%s", net.JoinHostPort(nodeIP, fmt.Sprint(controllerInfo.APIPort)))
 	return cfg, nil
 }
+
+type PodFileCopy interface {
+	CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error
+}
+
+type PodFile struct {
+	RestConfig *rest.Config
+	Client     kubernetes.Interface
+}
+
+func (p *PodFile) CopyFromPod(ctx context.Context, fs afero.Fs, namespace, name, containerName, srcPath, dstDir string) error {
+	dir, fileName := filepath.Split(srcPath)
```

Comment:
technically you should be using `path.Split` here and not `filepath.Split`, as the path is for a (potentially) different operating system